library/cwm/src/lib/cwm/abstract_widget.rb

```diff
@@ -83,6 +83,15 @@ def self.widget_type=(type)
     # @!method help
     #   @return [String] translated help text for the widget
 
+    # @!method dynamic_help
+    #   Dynamic help is useful when content of help text is not static and when
+    #   help is recomputed with {Yast::CWM#ReplaceWidgetHelp}, then it can
+    #   return different result. Examples where it is useful is e.g. replace point which
+    #   have dynamic content and different help text depending on what is displayed.
+    #   Another example is CWM::Table which can have dynamic header depending on conditions.
+    #   In those cases after call of {Yast::CWM#ReplaceWidgetHelp}, it will return new help text
+    #   @return [String] translated help text for the widget.
```

Comment:
Too many words. The docs should tell me: Do I need to implement this method? When?
```rb
#   @return [String] translated help text for the widget
#   If the help text for the widget may change over its lifetime, define `dynamic_help`
#   instead of `help`.
#   It will be called ...
```
 **uh oh**, I think if I implement this for a CustomWidget it will not help anyway
  because I would need to call ReplaceWidgetHelp myself , right?